launchable/commands/subset.py

```diff
@@ -199,18 +205,33 @@ def run(self):
                     res = client.request("post", path, data=gzip.compress(json.dumps(
                         payload).encode()), headers=headers, timeout=timeout)
                     res.raise_for_status()
-
                     output = res.json()["testPaths"]
                 except Exception as e:
                     if os.getenv(REPORT_ERROR_KEY):
                         raise e
                     else:
                         click.echo(e, err=True)
-                    click.echo(click.style("Warning: the service failed to subset. Falling back to running all tests", fg='yellow'), err=True)
-
+                    click.echo(click.style(
+                        "Warning: the service failed to subset. Falling back to running all tests", fg='yellow'), err=True)
 
             # regardless of whether we managed to talk to the service
             # we produce test names
-            click.echo(self.separator.join(self.formatter(t) for t in output))
+            if rest != "":
```

Comment:
A little surprising that the default value is empty string, not `None`!

Reply:
it is my miss understanding and test failed 🙇